app/src/main/res/layout/widget_toggle.xml

```diff
@@ -1,20 +1,110 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Widgets have a limited set of [Views](https://developer.android.com/guide/topics/appwidgets#CreatingLayout) that you can use, definitely not a ConstraintLayour :(